src/script/standard.cpp

```diff
@@ -35,22 +35,55 @@ const char* GetTxnOutputType(txnouttype t)
     return nullptr;
 }
 
-bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, std::vector<std::vector<unsigned char> >& vSolutionsRet)
+static bool MatchPayToPubkey(const CScript& script, valtype& pubkey)
 {
-    // Templates
-    static std::multimap<txnouttype, CScript> mTemplates;
-    if (mTemplates.empty())
-    {
-        // Standard tx, sender provides pubkey, receiver adds signature
-        mTemplates.insert(std::make_pair(TX_PUBKEY, CScript() << OP_PUBKEY << OP_CHECKSIG));
+    if (script.size() == 35 && script[0] == 33 && script[34] == OP_CHECKSIG) {
```

Comment:
CPubKey::COMPRESSED_PUBLIC_KEY_SIZE == 33, would add clarity

Reply:
Incidentally, I have #12461 to make these references more succinct.

Reply:
Done.